src/test/java/com/ioet/bpm/people/services/PersonServiceTest.java

```diff
@@ -50,4 +50,40 @@ public void whenEmailIsDifferentReturnTrue() {
         Person personToUpdate = Person.builder().authenticationIdentity("new@ioet.com").build();
         assertTrue(personService.emailChanged(oldPerson, personToUpdate));
     }
+
+    @Test
+    public void findPeopleByEmailUsesPersonRepository() {
+        Iterable<Person> personIterable = mock(Iterable.class);
+        Optional<Person> personOptional = Optional.of(mock(Person.class));
+        when(personRepository.findAll()).thenReturn(personIterable);
+        when(personRepository.findPersonByAuthenticationIdentity("some_email")).thenReturn(personOptional);
+        personService.findPeopleByEmail(null);
+        personService.findPeopleByEmail("some_email");
+        verify(personRepository, times(1)).findAll();
+        verify(personRepository, times(1)).findPersonByAuthenticationIdentity("some_email");
+    }
+
+    @Test
+    public void whenNullEmailIsPassedReturnIterable() {
+        Iterable<Person> personExpected = mock(Iterable.class);
+        when(personRepository.findAll()).thenReturn(personExpected);
+        Iterable<Person> personFound = personService.findPeopleByEmail(null);
+        assertEquals(personExpected, personFound);
+    }
+
+    @Test
+    public void whenEmailIsFoundReturnIterable() {
+        String email = "some_email";
+        Optional<Person> personExpected = Optional.of(mock(Person.class));
+        when(personRepository.findPersonByAuthenticationIdentity(email)).thenReturn(personExpected);
+        ArrayList<Person> personFound = (ArrayList<Person>) personService.findPeopleByEmail(email);
+        assertEquals(personExpected.get(), personFound.get(0));
+    }
+
+    @Test
+    public void whenEmailIsNotFoundReturnNull() {
```

Comment:
I think by now the name of that method needs to be changed

Reply:
:+1: